packages/nuxt/src/pages/runtime/router.ts

```diff
@@ -67,10 +67,23 @@ export default defineNuxtPlugin(async (nuxtApp) => {
   const router = createRouter({
     ...routerOptions,
     history,
-    routes
+    routes: []
   })
+  let clearRoutes = router.addRoute({ path: '', children: routes })
```

Comment:
shouldn't this be
```suggestion
  let clearRoutes = router.addRoute({ path: '/', children: routes })
```

did you find any trouble with it? The router is more preared to take paths that start with `/` when they have no parent